manage.html

```diff
@@ -254,7 +251,7 @@ <h1 id="manage-heading" i18n-text="manageHeading"></h1>
     <button id="check-all-updates-force" class="hidden" i18n-text="checkAllUpdatesForce"></button>
   </p>
 
-  <p>
+  <p class="border-bottom margin-bottom-0">
```

Comment:
NOOOOOOOO! 
give it a meaningful id liek "add-style-wrapper" or something and style it in css file